src/builtin.d

```diff
@@ -48,35 +49,35 @@ extern (C++) Expression eval_sin(Loc loc, FuncDeclaration fd, Expressions* argum
 {
     Expression arg0 = (*arguments)[0];
     assert(arg0.op == TOKfloat64);
-    return new RealExp(loc, sinl(arg0.toReal()), arg0.type);
+    return new RealExp(loc, TargetReal.sin(arg0.toReal()), arg0.type);
```

Comment:
And once again, that's where we disagree. I see no advantage by moving these helpers back to `Port`, see my arguments before. They **only** depend on the `real_t` type and don't just take and return one (they should actually be considered `pure`).